services/graph/pkg/service/v0/drives.go

```diff
@@ -468,19 +468,19 @@ func (g Graph) UpdateDrive(w http.ResponseWriter, r *http.Request) {
 		switch resp.Status.GetCode() {
 		case cs3rpc.Code_CODE_NOT_FOUND:
 			logger.Debug().Interface("id", rid).Msg("could not update drive: drive not found")
-			errorcode.ItemNotFound.Render(w, r, http.StatusNotFound, resp.GetStatus().GetMessage())
+			errorcode.ItemNotFound.Render(w, r, http.StatusNotFound, "drive not found")
 			return
 		case cs3rpc.Code_CODE_PERMISSION_DENIED:
 			logger.Debug().Interface("id", rid).Msg("could not update drive, permission denied")
-			errorcode.NotAllowed.Render(w, r, http.StatusForbidden, resp.GetStatus().GetMessage())
+			errorcode.ItemNotFound.Render(w, r, http.StatusNotFound, "drive not found")
```

Comment:
This should be "faked" on a deeper level. The CS3 API should already return the Item-Not-Found because there are different clients that use CS3 directly - and also should not get the information that the space exists.

Reply:
@dragotin What clients are you mean, mobile app, etc?

Reply:
@2403905 He means we should not fix that here on the graph service but on the original response from reva.

Reply:
@micbar Are we going to make such changes now? It leads to charges across all API.

Reply:
@dragotin We agreed with @butonic that we do not refactor the CS3 Api now.